pkg/model/bundle.go

```diff
@@ -63,6 +58,7 @@ type BundleEntry struct {
 	NameWithPath string      `json:"name" yaml:"name"`
 	FileMode     os.FileMode `json:"mode" yaml:"mode"`
 	Size         uint64      `json:"size" yaml:"size"`
+	SnapshotTime time.Time   `json:"snapshotTime,omitempty" yaml:"SnapshotTime,omitempty"` // time the file was uploaded
```

Comment:
was going to comment that we might need to take precautions for backwards-compatibility by adding some nil checks on this additional struct field.  except then i guess the `omitempty` token in the serialization dsl-in-annotations pretty much covers nil-checking?

while here, are the `omitempty` tags more or less current with those metadata fields that might be nil-able – like, are there any `omitempty` tags that can be removed supposing all metadata has been created by v2+?

Reply:
This extra field must only be present for "file list files" (aka index) produced by the split.
It is removed from the final bundle index files as I do not want to introduce any change with existing metadata.
Hence: `omitempty` does matter.